services/resource-usage-tracker/src/simcore_service_resource_usage_tracker/resource_tracker.py

```diff
@@ -23,6 +23,8 @@
 _TASK_NAME_START_PERIODIC_TASK = "start_background_task"
 _TASK_NAME_PERIODICALY_CHECK_RUNNING_SERVICES = "periodic_check_of_running_services"
 
+_RUT_MESSAGE_TTL_IN_MS = 2 * 60 * 60 * 1000  # 2 hours
```

Comment:
Similar suggestion here. 
```
_RUT_MESSAGE_TTL_IN_MS = 2 * _HOURS
```

Reply:
Q: why 2 hours? is this enough? last time it happened it was more like 1-2 days no?

Reply:
@sanderegg I put 2 hours as some reasonable number (for example usually our maintenance takes 2 hours). Regarding the problem from last time, even though it would be 1-2 days it would not help us as rabbitmq was not accessible. Also I am not sure whether it is a good idea to put some big number there I have not seen other people to put there too big number. One of the reasons might be memory utilization, but in our case I would say if rabbit is not accessible for more then 2 hours something is really bad and I am not sure whether we want to charge the user in that case.

Reply:
well then you need to be super sure that the maintenance never goes above 2 hours. Or, then you need to kill the computational jobs beforehand